deploy/backplane/srep/hypershift/rhobs-cluster/10-srep-management-cluster-project.ClusteRole.yml

```diff
@@ -0,0 +1,78 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-srep-rhobs-cluster-project
+rules:
+- apiGroups:
```

Comment:
Update resources to match what we need on the rhobs regional cluster. I'd suggest logging into the rhobs cluster and looking at CRD under the monitoring.rhobs api group.